src/com/qamp/HarisJasarevic/homeworks/homework2/task3/Robot.java

```diff
@@ -0,0 +1,74 @@
+package com.qamp.HarisJasarevic.homeworks.homework2.task3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Robot {
+
+    private int x;
+    private int y;
+    private int robotMoves;
+    private List<String> robotPositions;
```

Comment:
you are storing moves in a `List<String>` is there some better way to do it? 
Try to take inspiration from the other two tasks, Bookshelf for example.